client/components/site-selector/index.jsx

```diff
@@ -302,23 +307,27 @@ class SiteSelector extends Component {
 	}
 
 	renderAllSites() {
-		if ( ! this.props.showAllSites || this.props.sitesFound || ! this.props.allSitesPath ) {
+		const { allSitesPath, showAllSites, sitesFound, translate } = this.props;
+
+		if ( ! showAllSites || sitesFound || ! allSitesPath ) {
 			return null;
 		}
 
 		this.visibleSites.push( ALL_SITES );
 
-		const isHighlighted = this.isHighlighted( ALL_SITES );
+		const classes = classNames( 'all-sites', {
+			'is-selected': this.isSelected( ALL_SITES ),
+		} );
 
 		return (
-			<AllSites
+			<CompactCard
+				className={ classes }
+				href="/home"
 				key="selector-all-sites"
-				sites={ this.props.sites }
-				onSelect={ this.onAllSitesSelect }
 				onMouseEnter={ this.onAllSitesHover }
-				isHighlighted={ isHighlighted }
-				isSelected={ this.isSelected( ALL_SITES ) }
-			/>
+			>
+				{ translate( 'Manage All My Sites' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **56** times:
`translate( 'Manage all my sites' )` *ES Score: 10*